client/src/app/routes/Visualizer.tsx

```diff
@@ -189,7 +188,7 @@ class Visualizer extends Feeds<RouteComponentProps<VisualizerRouteProps>, Visual
     public render(): ReactNode {
         return (
             <div className={
-                classNames("visualizer", { "dark-mode": this.state.darkMode })
+                classNames("visualizer", { "dark-mode": this.props.darkMode })
```

Comment:
Leave only "visualizer" class name and remove "dark-mode" from class names as it is not used for switching to Dark mode.

Reply:
Resolved